R/makeClusterTask.R

```diff
@@ -79,10 +79,10 @@
 #' @import factoextra
 #' @export
 #'
-makeClusterTask = function(id, data, target, cluster.cols = NULL, method = "cluster.kmeans", random.seed = 89L,
+makeClusterTask = function(id, data, cluster.cols = NULL, method = "cluster.kmeans", random.seed = 89L,
```

Comment:
I removed target since it is not used in further calculation.
Note: Yes it ist used in getDataType but the information providet by this fun is not used

Reply:
ok. I will change that for the makeMDSTask as well